WST-backend/Assets/Scripts/Controller.cs

```diff
@@ -1,25 +1,70 @@
 using UnityEngine;
 using UnityEngine.InputSystem;
+using WST.Communication;
 
 namespace WST.Control {
     public class Controller : MonoBehaviour {
-        [SerializeField] private float speed = 10;
-        [SerializeField] private float throttle;
-        [SerializeField] private float yaw;
 
+        [Header("Input Settings")]
+        [SerializeField] private InputActionReference leftStickInput;
+        [SerializeField] private InputActionReference rightStickInput;
+
+        private float _virtualThrottle;
+        private float _virtualYaw;
+        private float _virtualPitch;
+        private float _virtualRoll;
+
+        private const float multiplier = 1000;
+
+        private DroneControlData controllData;
+        
         // TODO: Change name of 'SetRealLifeMoveValues' method
         // public void SetRealLifeMoveValues(InputAction.CallbackContext ctx) {
         //     if (ctx.performed) {
         //         moveDirection = ctx.ReadValue<Vector2>();
         //     }
         // }
 
+        public void Update()
+        {
+            Vector2 leftStickVal  = Vector2.zero;
+            Vector2 rightStickVal = Vector2.zero;
+
+            if (leftStickInput  != null) leftStickVal   = leftStickInput.action.ReadValue<Vector2>();
+            if (rightStickInput != null) rightStickVal  = rightStickInput.action.ReadValue<Vector2>();
```

Comment:
Is there really a point checking if `Vector2` is not null? It is quite expensive especially on every `Update` :/ Will the value ever be actually null or just equal to `Vector2.zero`?